tests/README.md

```diff
@@ -155,6 +155,15 @@ exit_code, output = ctrl.run_python("python/monitor.py")
 
 A full example of how to use the python bindings can be found in the [monitor open-close test](./tests/tier0/monitor-open-close/).
 
+### Generating test ID
+
```

Comment:
It would nice to add something like:

Each test should be clearly identified by unique ID, so when adding a new test, please run following command to assign ID to the new test: